OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -44,8 +44,13 @@ public class AirstrikePowerInfo : SupportPowerInfo
 
 	public class AirstrikePower : SupportPower
 	{
+		readonly AirstrikePowerInfo info;
```

Comment:
In practice it is guaranteed that the parent class's `Info` is an `AirstrikePowerInfo` to theres no need to cache this again here.  Better to cast it where you need it, which looks like only one extra place in this PR.

The better long term fix (for a different PR!) should be to redefine `SupportPower` as `public abstract class SupportPower<InfoType> : PausableConditionalTrait<InfoType> where InfoType : SupportPowerInfo` so that `Info` can be exposed directly as the correct type.

Reply:
I copied this pattern from `IonCannonPower` because I needed a way to use the info in more than one method. I am not sure how to go about fixing that (I'm still new to C#).

Reply:
I'll see if I can find time to fix this properly in the next couple of days so that you won't need to worry about it here.

Reply:
My idea for making `SupportPower` generic isn't compatible with the way it used in `SupportPowerManager`, so lets keep this part as-is.  We use this local-info pattern in several other support powers.